integration_tests/auth/conftest.py

```diff
@@ -21,7 +21,7 @@
 import pytest
 from _pytest.fixtures import SubRequest
 from faker import Faker
-from phoenix.auth import PHOENIX_ACCESS_TOKEN_COOKIE_NAME
+from phoenix.auth import DEFAULT_ENTROPY, PHOENIX_ACCESS_TOKEN_COOKIE_NAME
```

Comment:
while this variable name is conceptually accurate, I believe it is too obscure a connection to make for people unfamiliar with at least a little bit of information theory. Can we use something like `DEFAULT_SECRET_SIZE` or something similar instead?

Reply:
Sure, and just FYI, this is lifted from the Python standard library

https://github.com/python/cpython/blob/0181aa2e3efedc6504b27f6fe74f096e5e454286/Lib/secrets.py#L31

Reply:
I think the name makes sense, I just worry about using terminology that might be hard to connect to its use without domain knowledge